src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -333,9 +333,9 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin) {
     await this.$nextTick() // wait for form to update
 
     // if any component is invalid, don't save
-    const isNotationFormValid = this.$refs.notationFormRef.validate()
-    const isFileComponentValid = (!this.isCourtOrder || this.$refs.fileUploadRef.validate())
-    const isCourtOrderPoaValid = !this.isAdministerFreeze ? this.$refs.courtOrderPoaRef.validate() : true
+    const isNotationFormValid = (!this.$refs.notationFormRef || this.$refs.notationFormRef.validate())
+    const isFileComponentValid = (!this.$refs.fileUploadRef || this.$refs.fileUploadRef.validate())
+    const isCourtOrderPoaValid = (!this.$refs.courtOrderPoaRef || this.$refs.courtOrderPoaRef.validate())
```

Comment:
The above are each True if the component (ref) doesn't exist (and do not validate them).